docker-compose.yml

```diff
@@ -1,32 +1,34 @@
-portal:
-  build: .
-  env_file:
-    - .chameleon_env
-  volumes:
-    - ../media:/project/media
-    - ../static:/var/www/chameleoncloud.org/static
-    - .:/project
-  ports:
-    - 8888:8888
-  links:
-    - referenceapi:referenceapi
-  command: python manage.py runserver 0.0.0.0:8888
-referenceapi:
-  image: referenceapi:latest
-  ports:
-    - 8000:8000
-redis:
-  image: "redis:alpine"
-  expose:
-    - '6379'
-  ports:
-    - '6379:6379'
-celery:
-  build: .
-  command: celery -A chameleon worker -l info
-  env_file:
-    - .chameleon_env
-  volumes:
-    - .:/project
-  depends_on:
-    - redis
+version: '3.3'
+services:
+  portal:
+    build: .
+    env_file:
+      - .chameleon_env
+    volumes:
+      - ../media:/project/media
+      - ../static:/var/www/chameleoncloud.org/static
+      - .:/project
+    ports:
+      - 8888:8888
+    links:
+      - referenceapi:referenceapi
+    command: python manage.py runserver 0.0.0.0:8888
+  referenceapi:
+    image: referenceapi:latest
+    ports:
+      - 8000:8000
+  redis:
+    image: "redis:alpine"
+    expose:
+      - '6379'
+    ports:
+      - '6379:6379'
+  celery:
+    build: .
+    command: celery -A chameleon worker -l info --concurrency=1 -B
```

Comment:
The -B (--beat) has the worker run a beat scheduler.

Reply:
If I look up the [docs](https://docs.celeryproject.org/en/stable/reference/celery.bin.worker.html#cmdoption-celery-worker-b), there is a big note on this flag:

> -B is meant to be used for development purposes. For production environment, you need to start celery beat separately.

How difficult is it to start the beat separately? "for development purposes" can mean it's sort of OK if you're not expecting high load (which we're not), but it can also mean "we don't really test or support this in the same way, and it has these failure modes", which wouldn't be OK.